_source/_posts/2020-12-02-comparing-authentication-in-laravel-6-7-8.md

```diff
@@ -0,0 +1,202 @@
+---
+layout: blog_post
+title: "Comparing Authentication in Laravel versions 6, 7, and 8"
+author: karl-hughes
+by: contractor
+communities: [php]
+description: "Learn the differences between how Laravel versions 6, 7 and 8 handle authentication, everything from the UI to the new file structure."
+tags: [php, laravel, authentication]
+tweets:
+- "Learn how to upgrade authentication in your #Laravel 6 and 7 apps to the latest version 8!"
+- "Worrying about how updates in Laravel 8 might affect authentication in your app? Read this ➡"
+- "Learn the differences between how the three most recent #Laravel versions handle authentication"
+image: blog/featured/okta-php-headphones.jpg
+type: conversion
```

Comment:
This seems more like awareness than conversion since there's no Okta app created.